.github/scripts/start-backend.sh

```
@@ -0,0 +1,49 @@
+export CONFIG_PATH
+export GOODMAP_PATH
+make "$MAKE_TARGET" > "$LOG_FILE" 2>&1 &
+BACKEND_PID=$!
+echo $BACKEND_PID > "$PID_FILE"
+disown $BACKEND_PID
+
+# Wait for startup
+sleep "$STARTUP_WAIT"
+
+# Check if backend started successfully
+if ! kill -0 $(cat "$PID_FILE") 2>/dev/null; then
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)